src/retransmit/BUILD.gn

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/chip.gni")
+
+static_library("retransmit") {
+  output_name = "libRetransmit"
+
+  sources = [ "Cache.h" ]
+
```

Comment:
Could we throw `cflags = [ "-Wconversion" ]` in here so that this passes that check up front?